packages/java/engine-core/src/main/java/com/vaadin/hilla/engine/EngineConfiguration.java

```diff
@@ -129,8 +135,15 @@ public BrowserCallableFinder getBrowserCallableFinder() {
         return () -> {
             try {
                 return AotBrowserCallableFinder.findEndpointClasses(this);
-            } catch (IOException | InterruptedException e) {
-                throw new ExecutionFailedException(e);
+            } catch (Exception e) {
+                if (classFinder != null) {
+                    LOGGER.info(
+                            "AOT-based detection of browser-callable classes failed. Falling back to classpath scan.");
```

Comment:
Maybe we can improve the log message here so that it either:
1. instruct the user to run the build with DEBUG log level, so they get the info were to find the report (if they're interested).
2. to reveal the repot file path here directly, since the report is already there.

Reply:
Added a note to suggest using debug level. That will allow to discover the report file path.